src/ripple/app/tx/impl/ApplyContext.cpp

```diff
@@ -69,4 +71,57 @@ ApplyContext::visit (std::function <void (
     view_->visit(base_, func);
 }
 
+template<std::size_t... Is>
+TER
+ApplyContext::checkInvariantsHelper(TER terResult, std::index_sequence<Is...>)
+{
+    if (view_->rules().enabled(featureEnforceInvariants))
+    {
+        auto const& section = app.config().section("check_invariants");
+        if (get<bool> (section, "enabled", true))
+        {
+            auto checkers = getInvariantChecks();
```

Comment:
@JoelKatz would the command line option work for yoiu or is the config option still preferred? 

As for the existing implementation, it seems like having unlabeled values in a section is considered "legacy" behavior and that's why I avoided it. I don't have a strong opinion - I'll implement whatever our current config best practices are (if someone can clarify).  

If we do stick with some form of config option, I'll add it to the example config.